lib/eslint.js

```diff
@@ -184,11 +197,12 @@ function run(console, runOptions) {
         console.error("Analyzing: " + batchFiles)
       }
 
-      runWithTiming("analyze-batch-" + batchNum, function() {
-        batchReport = cli.executeOnFiles(batchFiles)
+      await runWithTimingAsync("analyze-batch" + batchNum, async function() {
```

Comment:
I noticed you change `analyze-batch-` to just `analyze-batch` is this intentional ?

Reply:
Not intentional, fixing